frontend/.gitignore

```diff
@@ -4,6 +4,9 @@
 /node_modules
 /.pnp
 .pnp.js
+package.json
+package-lock.json
```

Comment:
I think these two folders are needed to build your frontend e.g. in a GitHub action, therefore, you may want to consider not including them in your `.gitignore` as it may cause pipeline frontend builds to fail